src/style.css

```diff
@@ -0,0 +1,156 @@
+/* stylelint-disable-next-line no-empty-source */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  --main-text-color: #060b35;
+}
+
+.product {
+  font-family: "Roboto", sans-serif;
+  display: flex;
+  flex-direction: column;
+  border: rgb(245, 245, 245) solid 0.5px;
+  border-radius: 5px;
+  max-width: 199px;
```

Comment:
Change max-width to 200px and add box-sizing: border-box; rule, as the task description mentions the card width should be 200px including border